app/screens/LoginScreen.tsx

```diff
@@ -206,17 +211,18 @@ const LoginScreen = () => {
                 />
                 <View style={addressEntered ? undefined : AppStyles.noDisplay}>
                   <SpacerV />
-                  <H3 text={t("session.signing_command")}></H3>
+                  <H3 text={t(isSignCommand ? "session.signing_command" : "session.signing_message")}></H3>
                   <View style={[AppStyles.containerHorizontal, styles.signingMessage]}>
                     <View style={styles.textContainer}>
-                      <Text>{signingCommand(address)}</Text>
+                      <Text>{signMessage}</Text>
                     </View>
                     <SpacerH />
                     <IconButton
                       icon={signCommandCopied ? "check" : "content-copy"}
                       color={signCommandCopied ? Colors.Success : Colors.Grey}
                       onPress={() => {
-                        ClipboardService.copy(signingCommand(address));
+                        if (!signMessage) return;
+                        ClipboardService.copy(signMessage);
```

Comment:
btw, an even more concise way would be: 
(signMessage) && ClipboardService.copy(signMessage) 😉

and the even safer condition would be 
(signMessage != null) && ClipboardService.copy(signMessage)

Reply:
(early) returns, im allgemeinen, machen eigentlich nur bei Mehrzeiler wirklich Sinn

Reply:
it is more than a one liner 🤔 As we want to show the checkmark only, if a sign message was copied